_pages/platform/JWT-JWKS-authentication-org-management.md

```diff
@@ -89,4 +90,4 @@ For instance if the below token is supplied, and the organization is configured
 
 ![Example payload of a JWT](/developers/images/post_images/jwt-sync/jwt_payload.png)
 
-If you're new to Algorithmia and would like to learn more about our product and using JWT for authentication, please [contact our sales team](https://info.algorithmia.com/contact-sales). We'd love to hear from you!
\ No newline at end of file
+If you're new to Algorithmia and would like to learn more about our product and using JWT for authentication, please [contact our sales team](https://info.algorithmia.com/contact-sales). We'd love to hear from you!
```

Comment:
Maybe not worth an update, but in these cases I like using `mailto` links to make it easier for people to get in touch.
```suggestion
If you're new to Algorithmia and would like to learn more about our product and using JWT for authentication, please <a href="mailto:support@algorithmia.com?subject=I have a question about JWT authentication">contact our sales team</a>. We'd love to hear from you!
```